File: CheckMyStar.Bll.Models/AccommodationModel.cs
namespace CheckMyStar.Bll.Models
{
    public class AccommodationModel
    {
        public int Identifier { get; set; }
        public string? AccommodationName { get; set; }
        public string? AccommodationPhone { get; set; }
        public AccommodationTypeModel? AccommodationType { get; set; }
        public int? AccommodationCurrentStar { get; set; }
        public AddressModel? Address { get; set; }
        public bool IsActive { get; set; }
        public DateTime? CreatedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
    }
}

File: CheckMyStar.Angular/src/app/www/20_Models/BackOffice/Accommodation.model.ts
import { AccommodationTypeModel } from './AccommodationType.model'
import { UserModel } from '../Common/User.model';
import { AddressModel } from '../Common/Address.model';
import { FolderStatusModel } from './FolderStatus.model';

export interface AccommodationModel {
  identifier: number;
  accommodationName?: string;
  accommodationPhone?: string ;
  accommodationType?: AccommodationTypeModel;
  accommodationCurrentStar?: number;
  address?: AddressModel;
  isActive: boolean;
}